code-style.md

```diff
@@ -0,0 +1,78 @@
+# Chingu Coding Style
+- updated 11/4/18
+
+# Core
+- base of AirBnB coding style
+- readability and ease of writing
+- consistency across codebases and authors
+- rejecting rules that needlessly restrict coding
+
+# Basics
+- 2 spaces (not tabs)
+- semicolons to end all statements
+  - no inline statement chaining
+- maximum column length of 100 characters
+- new lines at the ends of each file
+
+# Trailing Commas
```

Comment:
JSX closing tags in new line
Same reasons as trailing commas in multi-line lists